docker-compose-files/ansible-compose/README.md

```diff
@@ -0,0 +1,99 @@
+### Ansible docker_service module
+
+Since docker_service depends on docker-compose >= 1.7.0, we need to use docker-engine and docker-py versions which are not yet available in CentOS or EPEL repositories.
+
+Before we get the latest versions of docker and docker-py, let's remove the existing packages on ADB.
+
+```bash
+sudo yum remove -y python-docker-py docker*
```

Comment:
I believe that this is something that should be executed inside CDK, right? If so, Can you include the vagrant ssh instructions? Thanks